test/test_helper.rb

```diff
@@ -1,8 +1,16 @@
-# Add simplecov
+require 'simplecov'
+SimpleCov.start
```

Comment:
It looks like you forgot to exclude your tests from coverage (this is just to clean up coverage output).

```suggestion
SimpleCov.start do
  add_filter 'test/' # Tests should not be checked for coverage.
end
```